packages/registrar_client/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@threefold/registrar_client",
+  "version": "0.1.0",
+  "description": "Registrar client for the ThreeFold Grid V4",
+  "keywords": [],
+  "author": "Salma Elsoly <salmaelsoly@gmail.com>",
+  "license": "ISC",
+  "main": "./dist/node/src/index.js",
+  "module": "./dist/es/src/index.js",
+  "exports": {
+    ".": {
+      "import": "./dist/es/src/index.js",
+      "require": "./dist/node/src/index.js"
+    }
+  },
+  "types": "./dist/es/src/index.d.ts",
+  "engines": {
+    "node": ">=20.10.0"
+  },
+  "directories": {
+    "lib": "src",
+    "test": "tests"
+  },
+  "files": [
+    "/dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/threefoldtech/tfgridv4-sdk-ts.git"
+  },
+  "scripts": {
+    "test": "jest",
+    "format": "prettier --write .",
+    "lint": "eslint src/**/*.ts",
+    "lint:fix": "eslint src/**/*.ts --fix",
+    "node-build": "tsc --build tsconfig-node.json",
+    "es-build": "tsc --build tsconfig-es.json"
+  },
```

Comment:
a command that build both es and node will be great
``json
 "build": "npm-run-all es6-build node-build",
``